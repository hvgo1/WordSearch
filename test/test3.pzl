VGCHJHNCKJKPCLQXHXWPROCESSINGHPJDMZBCDILFPNUZKBOAX
FZPSIFFBEJONWDHWWIGAEJLHGMXHGQMUGBEVKCZAGOLCCZPUCK
XONBEQYPOYAGUXLFJOUQRWJNVHRZLRAEVRCVXYSITPEIIASTYN
GGMVHAZFZKBOYOOGZIFMDYOYTPQDUXIBAHDIFCUGPCWCZPBPKJ
RSUFYEAMYQKNYDUEIELFUFEGAQZLCRIARIKXZTXSDCNNZGSUWL
RGBIVOAWGNMZADQHGWAZPHIVBHZTHWFNWNUCINPUTZKSNCATII
ZRVQXMTJEYECEYNRBITVDWUKGGWPEWOFZLAWPNDJELBKHEXAMM
MQCHCQTIACFEVUSDBWIOHLXXCRTWLOGZMKBMXTDWBROQHMTKTI
WJVFMBYPDLINECUBHKGPNPNBRUMDDVOZAKCSEFBNTOGQFPGUTA
DJMPPRDRCXLXWIRTTSIHKZTSAVEGCHSHRLKEHONDSFFCPKFUVQ
LPSBXQQSQQOHIBQMYUDRVYZECALPERUJJXKHPBYOMOUXPNXBXO
FLCJAQONIOYROMEMCUKJNZWLKGLYRFIGUCYTUKNPWUUWLQEKUN
IOOUNIDTWROFAUXWHCRAESAVUPJHAFTGXHUDAACENFTMBWWSGX
LQMCIGAEFVAWYLULRMPALXESHXMDWAEOOHVNMTRENNACSGZGXN
EPMZAMOUGELRMTPOCUUDGFNNPZHFDVNNZTYSASXWIYMMPHIJZX
EIAKENIEGCHRMIDXSQFWPCHSNIGERMRFDLXJHEYQODKZNAGDDZ
PWNWJGENSWEXPMQBCDXKTGAPFIZGAOEXYIFLDCRVZMEEBDOXHK
HDDJSOAXESQQEEQBURXBGONRQWWCHRTFIJZFDKKPDWWQIYKDJB
WWZJORAJROUOSDBPCAGLTMEHCYNPNINVDUJHJHCFJZNKTIBCYH
FNWSEQPZEFWNMIFBXOJYYLFIWZZJSGIVYZIKIHKYVCIBWHKAKC
YJVKEYIWETEGZAAPKBZJINTERACTIVEUACUHSUCTSWSMJQYFIG
EHFZATAKNWQTGLYVWYAQKSVNWPAMGSQIOLVJWFOBYQPATBLSRC
SDFMHPWOFATZABPTTEXTMAXOXBYRGFUJMGYQAMULRLMVWFVSJD
MDPMSFIZURBBTYVXSKCESVVYSBLVLNHIFONTLKCDEFTQLFFJLD
TBVTUCLOSEGIMPXWXTAECHGQLJGNKETBQXACBKVLKMGKVVXAMY
GLZIWFRJDGYUTUSGJNDRFVQMRWXAXBBRRRXKKVSYQGIAQPPVCP
SZWJTEOFDKICZDNTNYRTNOEPSHLWWPTZSELECTOPHIVSRFRTOA
SSQVRMGHUMDWBLBVEJDQSIHAXEGJKDQISSBXIBGWRTHWTPTQMO
SESZKFWYLQLLDQTBEWDDGJJHETYVXWLEWCPLLDBWGAYFGFQFLU
RBWEHFADQQOGWCPIXAJISBNPVFCIVEXBLXRFCHUDJMORDCCIUU
FYUGRVVHANFRBDIAWCPVPPQESSFBRFJEYTUPBPMZBRLPSIIKXP
TTVENDFTLJDUAWNCXNDDITVTYVCECUCSGFIWPFWQBONZNWRDAG
WGHURBNSESCCGBZMPVIZZLCBWOWTDRRATSTPMYTAQFHRPSDWGV
HPTNRIWNNEUDZLVXPKUMNNAUNLJLAFEBQQACIJCGMQCXRKZZUA
MOUSEPLWYDMJZISTAQUBHXSVJXMLMMVAUJPPTCPYASXMRLTTGV
JUDMQTKGAGOXUNKDXBZPENOJFAWXPQMTNDQHCVTOAGVHLUHJCY
OVNYFKOILTVYKSOLWVTUPASTEHUATJYAUEECRNQJKMOIKNHCKJ
WWETUPPSBWJTJJSMRETELEDQGQVOZOTDMOWWSSTUNEMCWXQWNE
KSMXKLBGZINSUKQGXNNODUZSRGOQMPXHELPIUOYUQKEPOWKLAA
LLEOCDERBHMNOQRSNBREAXMLLUVUJJBARLXAZELKESLADSIXNI
RJDCICDNNFAONETWORKHOCSEISHFFMHGIWYBZBCJBEULEJNEPO
YPOCMFYDCNOCHYRCIHPARGUOYRNMBRGQCSCVQMPOZXXZDFHKMR
GWMNDVFWRIJIFZYXORBJQNLSPRBWIENYHSEKPQPYSDOTBVIFRN
BNPWHYDNIFNXOIBKSFADEPLGUZKZXMXHQQZFGRBKHDKLXFFFTH
OPUUPOCODWTIELXBRMEWMZEAJVXURTWDJIABRJNJRNRVPEAOFE
NDSICDDLWPTFPQOJVXIJVBGGVFLXBFKQIVBNRPZFGVLTIEUFAO
MXYLSHZDYUCVODZUFXZAZMGRHJTWIWQNKLTFPWNHVBPVWDVJQH
CTZQWSXRYOSAHDLIVOKDTHOBDJWCFNRFJJXXXFPZGJJFAQATMS
OQSNCUHRPLRWOHNGVAICURXZHMFPERIPHERALAQYCRIUNLDESG
FKVPYQSAAVPTRGHVJASAHHQESRPRINTERRMPBHILTFTPTUXIUQ

cdrom
close
click
copy
cut
command
data base
delete
digital
file
find
font
format
graphic
icon
hardware
input
interactive
internet
keyboard
help
memory
menu
modem
mouse
multimedia
network
numeric
open
output
paste
peripheral
printer
processing
replace
save
scanner
search
select
software
text