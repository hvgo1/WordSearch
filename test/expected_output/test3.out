CDROM (46, 30) (42, 30)
CLOSE (6, 25) (10, 25)
CLICK (37, 30) (37, 26)
COPY (4, 42) (1, 42)
CUT (6, 25) (4, 25)
COMMAND (3, 12) (3, 18)
DATABASE (32, 38) (32, 31)
DELETE (23, 38) (18, 38)
DIGITAL (19, 11) (19, 5)
FILE (1, 12) (1, 15)
FIND (10, 44) (7, 44)
FONT (33, 24) (36, 24)
FORMAT (42, 33) (42, 28)
GRAPHIC (22, 42) (16, 42)
ICON (12, 43) (12, 40)
HARDWARE (29, 18) (29, 11)
INPUT (37, 6) (41, 6)
INTERACTIVE (21, 21) (31, 21)
INTERNET (31, 20) (31, 13)
KEYBOARD (18, 24) (18, 17)
HELP (32, 39) (35, 39)
MEMORY (16, 12) (11, 12)
MENU (43, 38) (40, 38)
MODEM (3, 43) (3, 39)
MOUSE (1, 35) (5, 35)
MULTIMEDIA (14, 12) (14, 21)
NETWORK (13, 41) (19, 41)
NUMERIC (33, 36) (33, 42)
OPEN (50, 41) (47, 41)
OUTPUT (48, 1) (48, 6)
PASTE (21, 37) (25, 37)
PERIPHERAL (28, 49) (37, 49)
PRINTER (27, 50) (33, 50)
PROCESSING (20, 1) (29, 1)
REPLACE (30, 11) (24, 11)
SAVE (24, 10) (27, 10)
SCANNER (45, 14) (39, 14)
SEARCH (22, 13) (17, 13)
SELECT (33, 27) (38, 27)
SOFTWARE (10, 18) (10, 25)
TEXT (17, 23) (20, 23)
